### ISSUE TITLE
Spike strip reassignment

### ISSUE THREAD

**Issue by JamStonedScones** [LondonsBlueLine/tracker#154]
Just a suggestion but making it so only RTPC can deploy spikes.

**Comment by Dale1987** [comment]
Only RTPC & RTPC subdivisions should have spikes anyway mate

**Comment by telboycafc** [comment]
Yeah this should already be a thing jam, but if there are any vehicles that don't have spikes in RTPC let us know.

**Comment by JamStonedScones** [comment]
My apologies, if that is the case feel free to remove this
